ui/app/components/transaction-list/transaction-list.component.js

```diff
@@ -40,7 +40,7 @@ export default class TransactionList extends PureComponent {
   shouldShowRetry = transaction => {
     const { transactionToRetry } = this.props
     const { id, submittedTime } = transaction
-    return id === transactionToRetry.id && Date.now() - submittedTime > 30000
+    return id === transactionToRetry.id && Date.now() - submittedTime > 3000
```

Comment:
Should this change be committed? Or was the time reduced to 3 seconds for testing purposes?